src/sentry/models/authenticator.py

```diff
@@ -164,6 +166,17 @@ class Meta:
         verbose_name_plural = _("authenticators")
         unique_together = (("user", "type"),)
 
+    def outboxes_for_update(self, shard_identifier: int | None = None) -> List[ControlOutboxBase]:
+        regions = find_regions_for_user(self.user_id)
+        return [
+            outbox
+            for outbox in OutboxCategory.USER_UPDATE.as_control_outboxes(
+                region_names=regions,
+                shard_identifier=self.user_id,
+                object_identifier=self.user_id,
+            )
+        ]
```

Comment:
What does this outer list comprehension add? It looks like it could be skipped/removed.